File: CSX/Functions and Execution Context/droids.js
function droids(arr) {
  let result = '';

  result = "These are not the droids you're looking for."

  for (let word of arr) {
    if (word == "Droids") {
      result = "Found Droids!"
    }
  }

  return result;
}
 
const starWars = ["Luke", "Finn", "Rey", "Kylo", "Droids"] 
const thrones = ["Jon", "Danny", "Tyrion", "The Mountain", "Cersei"] 
console.log(droids(starWars))
console.log(droids(thrones))

File: CSX - Python/Functions and Execution Context/droids.py
def droids(characters):
    for string in characters:
        if string == 'Droids':
            return 'Found Droids!'
    return "These are not the droids you're looking for."

star_wars = ['Luke', 'Finn', 'Rey', 'Kylo', 'Droids']
thrones = ['Jon', 'Danny', 'Tyrion', 'The Mountain', 'Cersei']

print(droids(star_wars))
print(droids(thrones))
